mcs/class/System.Drawing/Test/System.Drawing.Drawing2D/TestMatrix.cs

```diff
@@ -40,8 +40,8 @@ namespace MonoTests.System.Drawing.Drawing2D
 	public class MatrixTest {
 
 		private Matrix default_matrix;
-		private Rectangle rect;
-		private RectangleF rectf;
+		private Rectangle rect = default (Rectangle);
```

Comment:
Only purpose was to silence a compiler warning [CS0649 Field 'field' is never assigned to, and will always have its default value 'value'](https://msdn.microsoft.com/en-us/library/03b5270t.aspx). So not a strict requirement per se, I can revert this if you prefer.